index.js

```diff
@@ -0,0 +1,25 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let cards = document.querySelector(".card")
+    axios.get("http://localhost:3000").then(res => {  
+        let info = res.data[0]["results"][0]["results"]
+
+        for(let i = 0; i < info.length; i++){
```

Comment:
We should get into the habit of using forEach loops when parsing through json data